src/View.Uno.WinUI/View.Uno.WinUI.csproj

```diff
@@ -0,0 +1,53 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras/3.0.44">
```

Comment:
please make sure we have a trace somewhere saying we need to implement xamlmerge here

Reply:
Is there a PBI or feature to link in the mobile .NET committee backlog that I could link to the comment?

Reply:
@carlh98

Reply:
I've created them under the PBI for Nventive.View

Reply:
Add a comment with the link of the task here